account_banking_mandate/models/account_banking_mandate.py

```diff
@@ -35,6 +35,12 @@ def _get_states(self):
                 ('expired', 'Expired'),
                 ('cancel', 'Cancelled')]
 
+    format = fields.Selection(

```

Comment:
It doesn't add too much, because you're making an specific report for SEPA, independently from this field.